### PR TITLE
chore: rebase on latest Default Antora UI

### PR DESCRIPTION
**Pull request by zregvart**
This rebases the changes we made on top of the latest state of the
Default Antora UI. I've tweaked the bottom margin of the TOC, so that
the items do not render beneath the explore panel (panel with the
components/versions). And I've removed scrollbar customizations, I find
them gimmicky and they're negatively impacting the usability.

The major new feature is the new page table of content menu on the right
of each page.

### CODE REVIEW

**Comment by zregvart** [comment]
We need to discuss this on Monday's website call, perhaps have a round of (manual) testing before we merge this. Looks okay to me, but I could be missing something.

**Comment by zregvart** [comment]
cc @Delawen, @AemieJ, @aashnajena

**Comment by aashnajena** [comment]
I like the menu we're getting on the right. But I have concerns about smaller screens (as shown below). Need to investigate if this is due to tweaks we made, or is this a problem with Antora default UI? I can't really find any faults in the PR as such

image

**Comment by zregvart** [comment]
@aashnajena good catch, seems that we introduced that with #164, I think the solution would be to move the `aside` from `.doc` (in doc.css) to `.release` (in release.css). I should not have put that in the doc.css to begin with.

**Comment by djencks** [comment]
This problem looks closely related to Main content column shrinks to width of content <https://gitlab.com/antora/antora-ui-default/-/issues/134> which was fixed 4 weeks ago.  I’d suggest seeing what the page looks like with the actual default UI.  If this doesn’t happen with the default UI, then the camel customizations broke something.

thanks
David Jencks

**Comment by zregvart** [comment]
The TOC issue should be fixed with dfa6ea76f214333574458155b53abddce72e3371. Please have another look at the preview and share your feedback.

**Comment by aashnajena** [comment]
Looks good to me

**Comment by AemieJ** [comment]
@zregvart for the `mobile-navbar.js`, why is the section with dropdown deleted. As we haven't created the new design layout for the navbar menu, the dropdown items need to be shown.

File: antora-ui-camel/src/js/05-mobile-navbar.js [hunk 1]
```diff
@@ -9,12 +9,4 @@ document.addEventListener('DOMContentLoaded', function () {
       document.documentElement.classList.toggle('is-clipped--navbar')
     })
   })
-
-  var menuDropDowns = Array.prototype.slice.call(document.querySelectorAll('.has-dropdown'), 0)
```
[the remark this answers is not in the document]
Comment (aashnajena) [reply antora-ui-camel/src/js/05-mobile-navbar.js:13]: I'm able to see the dropdown menus, but I'm not able to toggle the navbar-burger icon on smaller screens. Although I think this piece of code was added for the menu-dropdowns in mobile menu

[the remark this answers is not in the document]
Comment (zregvart) [reply antora-ui-camel/src/js/05-mobile-navbar.js:13]: I think this should be fixed with  b7d1037

[the remark this answers is not in the document]
Comment (zregvart) [reply antora-ui-camel/src/js/05-mobile-navbar.js:13]: I still need to restore the dropdown menus though.

[the remark this answers is not in the document]
Comment (zregvart) [reply antora-ui-camel/src/js/05-mobile-navbar.js]: Sub-menus should be fixed in db8ce98. @aashnajena and @AemieJ can you take another look?

[the remark this answers is not in the document]
Comment (AemieJ) [reply antora-ui-camel/src/js/05-mobile-navbar.js]: Yes, LGTM now.

**Pull request merged**